docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -55,12 +55,12 @@ __Shorthand__: `-c`
 ### `--region`
 <p><small>| OPTIONAL | STRING |</small></p>
 
-__Description__: Sets your Sauce Labs region data center. Strongly recommended for best performance. Possible values are `us-west`, `eu-central`, `us-east`, and `apac-southeast`. For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
+__Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
```

Comment:
@waggledans I thought `us-west`, `eu-central`, `us-east`, and `apac-southeast` were the only possible values. Can you confirm what the others are?

Reply:
I'd rather not list them here. Those are dynamic values that may change over time

Reply:
Oh ok good thinking